bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/SvgImageCreatorImp.java

```diff
@@ -106,11 +146,11 @@ private boolean generateSvgImage(File imagePresentationDir, UploadedPresentation
 
             NuProcess process = convertImgToSvg.start();
             try {
-                process.waitFor(convPdfToSvgTimeout + 1, TimeUnit.SECONDS);
+                process.waitFor(convPdfToSvgTimeout, TimeUnit.SECONDS);

```

Comment:
```suggestion
                process.waitFor(convPdfToSvgTimeout + 1, TimeUnit.SECONDS);
```

I suggest to keep the extra 1 second.